pkg/runtime/wfengine/activity.go

```diff
@@ -184,6 +194,9 @@ loop:
 			if !t.Stop() {
 				<-t.C
 			}
+			// Activity execution failed with recoverable error. Record metrics
+			elapsed := diag.ElapsedSince(start)
```

Comment:
@shivamkm07 Please confirm that the activity execution completes even when a workflow is paused, just that a new activity in that workflow is not started when in pause state.

Reply:
@mukundansundar Confirmed that activity execution which has already started will complete even if a workflow is paused.

e.g. in Python-sdk, once workflow is paused, all future workflow events are buffered till the execution is resumed: https://github.com/microsoft/durabletask-python/blob/cbd07a8f2eae94ba82de798341d7c14f53b34693/durabletask/worker.py#L507
But this doesn't stop an activity from executing: https://github.com/microsoft/durabletask-python/blob/cbd07a8f2eae94ba82de798341d7c14f53b34693/durabletask/worker.py#L746

Reply:
should I resolve this conversation then ?